Nos/Service/CrashReporting.swift

```diff
@@ -45,10 +46,12 @@ class CrashReporting {
     }
 
     func report(_ error: Error) {
+        Log.info("Reporting error to Crash Reporting service: \(error.localizedDescription)")
```

Comment:
Can you make these `error`-level logs instead of `info`? We might have `Log.error(...)` but if not we have the poorly-named `Log.optional(...)` which log it as an error.

Reply:
Yep, I went with `error` rather than `optional`.